conf/application.conf

```diff
@@ -34,9 +34,7 @@ application.langs="en"
 # By convention, the default datasource is named `default`
 #
 db.default.driver=org.postgresql.Driver
-db.default.url="jdbc:postgresql://localhost:5432/mp_dev"
-db.default.username=osm
-db.default.password=osm
+db.default.url=${DATABASE_URL}
```

Comment:
@Crashfreak note the removal of the DB defaults from the config file. As noted in the PR comment you now need an envvar like `DATABASE_URL=jdbc:postgresql://osm:osm@localhost:5432/mp_dev`. This is required to make the Heroku build work, and I prefer not to have credentials in the config file anyway.